looty/src/main/scala/looty/poeapi/PoeTypes.scala

```diff
@@ -139,19 +139,35 @@ object PoeTypes {
       val currency = FrameType(5, "currency", "rgb(165,146,99)")
       val divCard = FrameType(6, "card", "rgb(165,146,99)")
       val quest = FrameType(7, "quest", "rgb(0,255,0)")
+      val prophecy = FrameType(8, "prophecy", "rgb(128,128,0)")
+      val relic = FrameType(9, "relic", "rgb(64,192,64)")
 
-      val all = Array(normal, magic, rare, unique, gem, currency, divCard, quest)
+      val all = Array(normal, magic, rare, unique, gem, currency, divCard, quest, prophecy, relic)
     }
 
-    val vaalFragments = ISet(
+    val mapFragments = ISet(
       "Sacrifice at Dusk",
       "Sacrifice at Midnight",
       "Sacrifice at Dawn",
       "Sacrifice at Noon",
       "Mortal Grief",
       "Mortal Rage",
       "Mortal Hope",
-      "Mortal Ignorance")
+      "Mortal Ignorance",
+      "Offering to the Goddess",
+      "Volkuur's Key",
+      "Yriel's Key",
+      "Eber's Key",
+      "Inya's Key",
+      "Fragment of the Minotaur",
+      "Fragment of the Chimera",
+      "Fragment of the Phoenix",
+      "Fragment of the Hydra",
+      "Ancient Reliquary Key")
```

Comment:
hmm i wonder if there is something common to map fragments that isn't in other items that we could use instead of listing them all out?